packages/core/src/Stores/ui-store.js

```diff
@@ -550,8 +550,9 @@ export default class UIStore extends BaseStore {
     }
 
     @action.bound
-    toggleAccountTransferLimitModal(state_change = !this.is_account_transfer_limit_modal_visible) {
+    toggleAccountTransferLimitModal({ state_change = !this.is_account_transfer_limit_modal_visible, message }) {
         this.is_account_transfer_limit_modal_visible = state_change;
+        this.account_transfer_limit_message = message;
```

Comment:
IMO we shouldn't tie our UI store to API error messages; it would be better if the transfer limit modal itself was in cashier and this was also in cashier-store since that's where it's used. but the file is not in cashier, so perhaps that should be done in a separate card.

Reply:
@akmal-binary , then let's move the `toggleAccountTransferLimitModal` to `cashier-store` instead?